aii-core/src/main/perl/Shellfe.pm

```diff
@@ -859,6 +885,31 @@ sub check_protected {
     return %hash;
 }
 
+sub change_dhcp 
+{
+    my ($self, $method, %nodes) = @_;
+    $self->debug(5,"logfile:", $self->option('logfile'), " dhcpcfg:", $self->option('dhcpcfg') );
+    my $dhcpmgr = AII::DHCP->new('script',
```

Comment:
@stdweird  ah yes, this is something i still need to figure out. I saw this used in the existing tests. Without this I can't seem to pass the parameters.